docs/ADDING_MACHINES.md

```diff
@@ -327,12 +327,12 @@ remember that the top, side and bottom textures of a casing must be `modern_indu
 For example:
 ```js
 MIMachineEvents.registerCasings(event => {
-    // Register two casings.
+    // Register a casing.
     // This doesn't register any model! Either add models or add the top/side/bottom textures.
-    event.register("my_fancy_casing", "my_other_casing");
+    event.register("fancy_casing", "Fancy");
```

Comment:
This is a breaking change for KubeJS users, might be best to avoid it?

Reply:
Not sure what would be a good solution for this, the translation name for casings are used in the tooltips of blocks in the structure PR I've been working on.